enforcer/lookup/lookup.go

```diff
@@ -139,23 +140,35 @@ func (m *PolicyDB) AddPolicy(selector policy.TagSelector) (policyID int) {
 
 }
 
+func (m *PolicyDB) keyValueFromString(tag string) (key, value string) {
+
+	parts := strings.SplitN(tag, "=", 2)
```

Comment:
It shouldnt be 2 but -1. Basically, I dont think we should allow = in value otherwise we can misinterpret it.